src/inverseRobot.js

```diff
@@ -23,8 +23,23 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  for (let i = 0; i < Object.values(robot).length - 1; i++) {
+    for (let ii = i + 1; ii < Object.values(robot).length; ii++) {
+      if (Object.values(robot)[i] === Object.values(robot)[ii]) {
+        return null;
+      }
+    }
+  }
+
+  const newRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
```

Comment:
```suggestion
  for (const key in robot) {
```
Тебе хватит только key, не надо создавать массив